ghost/core/core/server/services/lib/MailgunClient.js

```diff
@@ -106,7 +106,11 @@ module.exports = class MailgunClient {
 
             const mailgunConfig = this.#getConfig();
             startTime = Date.now();
-            const response = await mailgunInstance.messages.create(mailgunConfig.domain, messageData);
+
+            // Use overriden domain if specified in message
+            const mailDomain = message.domain ? message.domain : mailgunConfig.domain;
```

Comment:
Related to comment above. This makes it clear, but when reading it originally, I'm wondering why we would pass an `undefined` as `domain`.